skatepark/sass/base/_text.scss

```diff
@@ -1,4 +1,12 @@
 // Text selection text color
 ::selection {
 	color: var(--wp--preset--color--background);
+}
+
+.wp-block-column, .wp-block-group {
```

Comment:
These styles might better belong if blockbase, but I'm not sure about this idea. 

My reasoning for adding it is — I don't think that the first piece of text should have a top margin when contained in a column or group, because it ends up stacking if there is a padding declared. But maybe there is a better way of targeting this.

Reply:
I'm not sure we would do this for every theme, it feels kind of opinionated, it makes sense for Skatepark because of the border but that's it. When I first saw the pattern using the border I was thinking about it having a block style even but this approach makes sense too, but just in the scope of this specific theme.

Reply:
> ... it makes sense for Skatepark because of the border but that's it.

Since Blockbase [opts into border controls](https://github.com/Automattic/themes/blob/85249e67b98fc7b8d2b40521c17bee49b7ae799f/blockbase/theme.json#L32-L37), there's a UI for users to create this bordered effect on any Group block though — for Blockbase or for any of its children. 

I don't love these rules (and I hope Gutenberg adds greater per-block margin controls so folks can manually fix this sort of thing in the editor), but I think if we add these, they should be in Blockbase.

Reply:
Also, putting them in Blockbase would mean more universal expectations around this behavior for our themes, and also would make it easier to strip this out if it's not needed in the future. (Or if we come up with a better solution)

Reply:
yeah, I see what you mean, I don't like it but I agree, and having it on Blockbase is easier to control and go back to it when it eventually can be done better. I just hope we don't have to do this too much, Blockbase is growing instead of shrinking.